packages/graphql-toolbox/src/index.css

```diff
@@ -165,3 +165,11 @@
 }
 
 /* ----------------------*/
+
+.hover-show {
```

Comment:
The name suggests something to be visible/shown but the CSS specified does the opposite.